community/ndp/kernelextension/pom.xml

```diff
@@ -5,10 +5,10 @@
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
-    <groupId>org.neo4j</groupId>
-    <artifactId>parent</artifactId>
+    <groupId>org.neo4j.ndp</groupId>
+    <artifactId>neo4j-data-protocol-parent</artifactId>
```

Comment:
Hmm, I hadn't seen that. Fair enough. I don't feel very strongly about it, and it's more intuitive that the parent directory also has the direct parent. 

Also, this is all kind of related to the devteam mail about module structure. Happy to merge this as-is.